provisioning/templates-api/kubernetes/templates/api/deployment.yaml

```diff
@@ -37,8 +37,9 @@ spec:
           ports:
             - containerPort: 8000
           env:
+            # 80% from the resources.limits.memory
             - name: GOMEMLIMIT
-              value: "512MiB"
+              value: "400MiB"
```

Comment:
`GOGC=100` je default value, a znamena, ze garbage collector sa spusti, ked sa objem alokovanej pamate zdvojnásobí (narastie o `100%`), napr. `4kB, 8kB, 256MiB, 512Mib`.

`GOMEMLIMIT` hovori Go, ze ked sa blizi k tomuto limitu, tak ma GC spustat castejsie.
GC spomaluje apku, takze sa ho snazi Go volat co najmenej.

Ked sme mali tento limit na `100%`, tak sme zbytocne dostavali warnings.
Povieme Go, aby neslo cez `80%`, teda GC sa bude volat trocha castejsie (nemalo by to mat meratelny vplyv).